OSRMTextInstructions/OSRMTextInstructions.swift

```diff
@@ -276,6 +276,7 @@ public class OSRMInstructionFormatter: Formatter {
                 if let tokenType = TokenType(description: token! as String) {
                     var replacement: String
                     switch tokenType {
+                    case .code: replacement = step.codes?.first ?? ""
```

Comment:
Note that instructions coming from the .json never contain `{code}`. We’re inventing `TokenType.code` for the purpose of those format strings above that combine the way name and ref together into the `wayName` string, handled on the line below.